src/rsysreqs/scripts/scripts_ubuntu.go

```diff
@@ -0,0 +1,14 @@
+package scripts
+
+import "fmt"
+
+type ubuntuScriptGenerator struct{}
+
+func (g ubuntuScriptGenerator) InstallScripts(packages []string) []string {
+	var scripts []string
+	for _, pkg := range packages {
+		script := fmt.Sprintf("apt-get install -y %s", pkg)
+		scripts = append(scripts, script)
+	}
+	return scripts
+}
```

Comment:
Not sure if this needs to include an update command (e.g. `apt-get update -qq`) but that could be added later if so